scrypto-unit/src/test_runner.rs

```diff
@@ -227,6 +235,22 @@ impl TestRunner {
         }
     }
 
+    pub fn get_snapshot(&self) -> TestRunnerSnapshot {
```

Comment:
Minor: rename to `create_snapshot()`. `get_` seems to imply to me a return of borrowed value.

Reply:
Yup, that makes sense.